grails-app/domain/com/unifina/domain/data/Stream.groovy

```diff
@@ -9,11 +9,19 @@ import grails.converters.JSON
 import groovy.transform.CompileStatic
 
 class Stream implements Comparable {
+	public final static Integer DEFAULT_PARTITIONS = 1
 	public final static String DEFAULT_NAME = "Untitled Stream"
+	public final static boolean DEFAULT_UI_CHANNEL = false
+	public final static boolean DEFAULT_INBOX = false
+	public final static boolean DEFAULT_REQUIRE_SIGNED_DATA = false
```

Comment:
Hmm, isn't this unnecessary boilerplate? Was there a problem with giving the default values on variable introduction?

Reply:
@hpihkala No problem, just thought that it could be useful in unit tests, but then again maybe not.